manifests/enterprise.pp

```diff
@@ -0,0 +1,82 @@
+# Class splunk::enterprise
+#
+class splunk::enterprise (
+  String $version                          = $::splunk::params::version,
```

Comment:
for all the strings, can you update it to `String[1]` to prohibit empty strings?

Reply:
sure can!